manage_user_page.php

```diff
@@ -170,7 +170,7 @@
 	} else {
 		$t_title = '';
 	}
-	$t_active = $t_prefix === $f_filter ? 'active' : '';
+	$t_active = (string)$t_prefix === $f_filter ? 'active' : '';
```

Comment:
It is strange that when the array key and value were set earlier they were already casted to a string.  Do you understand why this happens?  Is there a reason we have '===' specially if we are going to cast anyway?